.github/workflows/ci.yml

```diff
@@ -62,15 +62,15 @@ jobs:
         run: gcc test/test-stack.c -o a.out && ./a.out
 
       - name: Generate parser
-        if: ${{ runner.os == 'macOS' }}
+        if: ${{ runner.os != 'Windows' }}
```

Comment:
Do we know that the parser is freshly generated for Linux, and now using whatever that's already there?

Reply:
I think with your recent PR bringing down memory usage we can assume it _is_ being generated and then revert if it's not the case?

Reply:
Looking at the time difference now I'm assuming that generation on linux _is_ happening since before it was taking like < 20 second and now 2 minutes.